controllers/manager/manager.go

```diff
@@ -444,6 +464,38 @@ func (m *PlatformManager) GetPlatformClient(namespace string) *gophercloud.Servi
 	return nil
 }
 
+func (m *PlatformManager) SendHostStrategyUpdate(host_strategy HostStrategyInfo) error {
```

Comment:
Need to add description on every new methods, difficult to read & understand

Reply:
> Need to add description on every new methods, difficult to read & understand

Done.